exercises/etl/canonical-data.json

```
@@ -1,19 +1,22 @@
 {
   "exercise": "etl",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "cases": [
     {
       "comments": [
+        "Transforms a set of legacy scrabble data stored as letters per score",
+        "to a set of data stored score per letter."
```

I think a comma is missing here.

```suggestion
        "to a set of data stored score per letter.",
```